governance/program/src/instruction.rs

```diff
@@ -234,6 +238,25 @@ pub enum GovernanceInstruction {
         signatory: Pubkey,
     },
 
+    /// Adds a required signatory to the Governance, which will be applied to all proposals created with it
+    ///
+    ///   0. `[writable, signer]` The Governance account the config is for
+    ///   1. `[writable]` RequiredSignatory Account
+    ///   2. `[signer]` Payer
+    ///   3. `[]` System program
+    AddRequiredSignatory {
```

Comment:
`AddRequiredSignatory` must be added at the end of the struct for backward compatibility of the instructions ids

Reply:
Resolved, although removing RemoveSignatory broke backwards compatibility anyway

Reply:
Correct, this is why you can't remove `RemoveSignatory` from the enum. It should be renamed to `Legacy1` to keep it backward compatible